app/src/main/java/org/metabrainz/mobile/api/webservice/MusicBrainzWebClient.java

```diff
@@ -1,17 +1,5 @@
 package org.metabrainz.mobile.api.webservice;
 
```

Comment:
Remove apache classes to generate apk. Keep copy of code in comments for future reference.